chainercv/transforms/bbox/non_maximum_suppression.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import division
+import numpy as np
+
+
+def non_maximum_suppression(bbox, threshold, limit=None, return_param=False):
+    """Suppress bounding boxes according to their Jaccard overlap.
+
+    This method checks each bounding box sequentially and selects the box
+    if the Jaccard overlap between the box and previously selected boxes
+    is less than :obj:`threshold`. This method is mainly used as postprocessing
+    of object detection. In this case, the bounding boxes should be sorted in
+    descending order.
+
+    The bounding boxes are expected to be packed into a two dimensional
+    tensor of shape :math:`(R, 4)`, where :math:`R` is the number of
+    bounding boxes in the image. The second axis represents attributes of
+    the bounding box. They are :obj:`(x_min, y_min, x_max, y_max)`,
+    where the four attributes are coordinates of the bottom left and the
+    top right vertices.
+
+    Args:
+        bbox (~numpy.ndarray): Bounding boxes to be augmented. The shape is
+            :math:`(R, 4)`. :math:`R` is the number of bounding boxes.
+        threshold (float): Thresold of Jaccard overlap.
+        limit (int): the upper bound of the number of selected boxes. If it is
+            not specified, this method selects as many boxes as possible.
+        return_param (bool): returns information of selection.
+
+    Returns:
+        ~numpy.ndarray or (~numpy.ndarray, dict):
+        If :obj:`return_param = False`,
+        returns an array :obj:`out_bbox` that is the result of suppression.
+
+        If :obj:`return_param = True`,
+        returns a tuple whose elements are :obj:`out_img, param`.
+        :obj:`param` is a dictionary of intermediate parameters whose
+        contents are listed below with key, value-type and the description
+        of the value.
+
+        * **selection** (:obj:`~numpy.ndarray`): An array which indicates each\
```

Comment:
It is consistent with the style of parameters to write type as italic letters.
Currently, I have not figured out a way to add a link to the word.

I chose visual consistency over the utility to have links.